tests/utils/setup_test.go

```diff
@@ -14,6 +15,60 @@ import (
 	. "github.com/kedacore/http-add-on/tests/helper"
 )
 
+var (
+	OtlpConfig = `mode: deployment
+config:
+  exporters:
```

Comment:
```suggestion
image:
  repository: "otel/opentelemetry-collector-contrib"
config:
  exporters:
```

Reply:
There is a potential breaking change that requires the change above, please commit it 🙏 
https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-collector/UPGRADING.md#0880-to-0890

Reply:
Done @JorTurFer :) 

Also, I've pushed a change to move the pending request count from a Gauge to a UpDownCounter (noticed some memory issues with the Gauge). Left it in a separate commit rather than squashing locally so it's easier for you to review.